pkg/apis/pipeline/v1alpha1/verificationpolicy_types.go

```diff
@@ -90,6 +90,13 @@ type KeyRef struct {
 	// Data contains the inline public key.
 	// +optional
 	Data string `json:"data,omitempty"`
+	// KMS contains the KMS url of the public key
+	// Supported formats differ based on the KMS system used.
+	// One example of a KMS url could be:
```

Comment:
Are there docs you could link to here instead?

Reply:
There are docs in Chains, I remember David suggest adding an example here to show how does a sample kms url look like.

Reply:
Is sigstore the authority on what's a valid KMS URL? or do you have to look at gcp, aws, azure, etc? Can you say something like "see sigstore/your cloud provider KMS docs for URL specification" along with the example you have here?

Reply:
Oh yes, I found one in sigstore docs https://docs.sigstore.dev/cosign/kms_support. Is this ok?

Reply:
I added this link in the comment

Reply:
Great that's perfect